pkgs/top-level/static.nix

```diff
@@ -93,27 +93,36 @@ in {
   clangStdenv = foldl (flip id) super.clangStdenv staticAdapters;
   libcxxStdenv = foldl (flip id) super.libcxxStdenv staticAdapters;
 
-  zlib = super.zlib.override {
-    # Don’t use new stdenv zlib because
-    # it doesn’t like the --disable-shared flag
+  boost = super.boost.override {
+    # Don’t use new stdenv for boost because it doesn’t like the
+    # --disable-shared flag
     stdenv = super.stdenv;
   };
+
+  curl = super.curl.override {
+    brotliSupport = false;
```

Comment:
Maybe we should comment this.

Reply:
@SuperSandro2000, I added a comment about the disabled brotli. Ready to merge?